torch/_prims/__init__.py

```diff
@@ -1918,8 +1932,9 @@ def _full_aten(
     device: torch.device,
     requires_grad: bool,
 ) -> Tensor:
+    # Note that Mypy thinks torch.full can't accept a complex fill_value
```

Comment:
That just means full's pyi annotation is incorrect, need to be generalized a little then